src/Common/src/Interop/Windows/kernel32/Interop.Access.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+		/// <summary>
+        /// <a href="https://docs.microsoft.com/en-us/windows-hardware/drivers/kernel/buffer-descriptions-for-i-o-control-codes">RequiredAccess</a>.
```

Comment:
Why this link?

Reply:
Thank you for catching this, must have mixed up the links. I have updated the link with the second link, since the first link throws error '404 Content not found'